README.md

```diff
@@ -42,12 +42,37 @@ More information about Black, you find at
 The following software is required to run PTF:
 
  * Python 2.7 or 3.x
- * Scapy 2.4.4
+ * Scapy 2.4.5
+ * isort 1.16.0
```

Comment:
seems unrelated to this change, and it would be a dev requirement anyway, not a requirement to run PTF itself

Reply:
It should be `six` not `isort`, corrected.